CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ## Neste versjon
 
+- ✨ **Infomasjonsbehandling**. Medlemmer kan nå velge studieretningen BAINFO når de registrer seg.
```

Comment:
Greit å legge til at drift ble fjernet i tillegg

Reply:
De er ikke fjernet fra nettsiden. Tidligere driftere kan fremdeles registrere seg, vi fjerner bare muligheten til å prioritere dem.

Reply:
Ja dårlig ordvalg fra min side, men forøvrig ikke fjernet dem, men changeloggen er grei for oss å se i fremtiden også så tenkte det kunne være greit å få dokumentert at det var på eventuelt den versjonen at mye av funksjonaliteten til Drift ble fjernet.

Reply:
Ser den, oppdaterte nå